virtualcluster/pkg/syncer/conversion/equality.go

```diff
@@ -19,7 +19,7 @@ package conversion
 import (
 	"strings"
 
-	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 
```

Comment:
you can remove line there, and re-sort imports.

Reply:
Done 👍